bundler.go

```diff
@@ -321,16 +348,15 @@ func (b *Bundler) bundle(e ConfigurationEnvironment) (err error) {
 		return
 	}
 
+	fmt.Println("initial ldflags", b.ldflags.String())
 	// Build ldflags
-	var l = ldflags{
-		"s": []string{},
-		"X": []string{
-			`"main.AppName=` + b.appName + `"`,
-			`"main.BuiltAt=` + time.Now().String() + `"`,
-		},
-	}
+	b.ldflags["s"] = []string{}
+	b.ldflags["X"] = append(b.ldflags["X"],
```

Comment:
Instead of modifying values of `b.ldflags` like this, I would leave the created `l` variable and add a `merge` method to the `LDFlags` struct that takes another `LDFlags` as input and merges its values.

Does that make sense?

Reply:
sure, I'll try to implement a merge method later today.